src/datasources/GenerateCollectionVariablesAPI.js

```diff
@@ -0,0 +1,40 @@
+import { RESTDataSource } from '@apollo/datasource-rest'
+import { camelCase, mapKeys } from 'lodash'
+
+export class GenerateCollectionVariablesAPI extends RESTDataSource {
+  constructor(host, token) {
+    super()
+    this.baseURL = `http://${host}`
```

Comment:
This should almost certainly be `https` right? I can see for localhost, but I don't see where a fallback would be using a https.